homeassistant/components/netio/switch.py

```diff
@@ -166,9 +166,9 @@ def turn_off(self, **kwargs: Any) -> None:
         self._set(False)
 
     def _set(self, value):
-        val = list("uuuu")
+        val = "uuuu"
```

Comment:
Removing `list` introduces a bug, as the next line assumes it can set individual characters by index.

Reply:
Done